### ISSUE TITLE
[Bug] Activity played time is missing

### ISSUE THREAD

**Issue by maicol07** [SpikeHD/rsRPC#9]
Activity played time tracking is missing from the activity in Discord, but was working in 0.21 and works in arRPC (time is displayed on Discord)

**Comment by SpikeHD** [comment]
Is this with a detected process or with an activity that uses IPC?

**Comment by maicol07** [comment]
I've tested it only with an IPC activity

**Comment by SpikeHD** [comment]
Which IPC activity? (preferably a game, or one I can test easily)

**Comment by maicol07** [comment]
I've tried both [VSCode extension](https://github.com/leonardssh/vscord) and [CustomRP](https://github.com/maximmax42/Discord-CustomRP) but they don't show any time.
If I set a manual timestamp in CustomRP then the time shows but it's stuck at 00:00

**Comment by SpikeHD** [comment]
What is the log output from rsRPC when trying these?

**Comment by maicol07** [comment]
Here you go:
```
 Maicol@MAICOL-PC ~\Downloads  .\rsrpc.exe -d .\detectable.json
[2024-12-22 01:49:56] [IPC] Created IPC socket: \\.\pipe\discord-ipc-0
[2024-12-22 01:49:56] [Websocket] Server started on port 6463
[2024-12-22 01:49:56] [RPC Server] Starting client connector on port 1337...
[2024-12-22 01:49:56] [RPC Server] Starting IPC connector...
[2024-12-22 01:49:56] [RPC Server] Starting process server...
[2024-12-22 01:49:56] [RPC Server] Starting websocket connector...
[2024-12-22 01:49:56] [RPC Server] Done! Watching for activity...
[2024-12-22 01:49:56] [Websocket] Polling for events...
[2024-12-22 01:49:56] [Process Scanner] Process scan triggered
[2024-12-22 01:50:02] [Process Scanner] Process scan complete
[2024-12-22 01:50:02] [Client Connector] No clients connected, skipping
[2024-12-22 01:50:05] [IPC] Incoming stream...
[2024-12-22 01:50:05] [IPC] Recieved message: {"v":1,"client_id":"1070048358246387864"}
[2024-12-22 01:50:05] [IPC] Recieved handshake
[2024-12-22 01:50:11] [Client Connector] Client 0 connected
[2024-12-22 01:50:12] [Process Scanner] Process scan triggered
[2024-12-22 01:50:19] [Process Scanner] Process scan complete
[2024-12-22 01:50:19] [Client Connector] Sending payload for activity: DSX
[2024-12-22 01:50:29] [Process Scanner] Process scan triggered
[2024-12-22 01:50:35] [Process Scanner] Process scan complete
[2024-12-22 01:50:35] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:50:45] [Process Scanner] Process scan triggered
[2024-12-22 01:50:50] [IPC] Incoming stream...
[2024-12-22 01:50:51] [IPC] Recieved message: {"v":1,"client_id":"810516608442695700"}
[2024-12-22 01:50:51] [IPC] Recieved handshake
[2024-12-22 01:50:52] [Process Scanner] Process scan complete
[2024-12-22 01:50:52] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:50:53] [IPC] Recieved message: {"cmd":"SET_ACTIVITY","args":{"pid":137344,"activity":{"details":"In no workspace - 0 problems found","state":"Working on configs.user.ini:19:23","instance":true,"assets":{"large_image":"https://raw.githubusercontent.com/LeonardSSH/vscord/main/assets/icons/manifest.png","small_image":"https://raw.githubusercontent.com/LeonardSSH/vscord/main/assets/icons/vscode.png","large_text":"Editing a MANIFEST file","small_text":"Visual Studio Code"},"timestamps":{"start":1734828652178}}},"nonce":"139a2888-7ede-4f42-8641-c11ea58c70e9"}
[2024-12-22 01:50:53] [Client Connector] Sending payload for IPC activity: "{\"activity\":{\"timestamps\":{\"start\":1734828652178000},\"application_id\":\"810516608442695700\",\"details\":\"In no workspace - 0 problems found\",\"state\":\"Working on configs.user.ini:19:23\",\"assets\":{\"large_image\":\"https://raw.githubusercontent.com/LeonardSSH/vscord/main/assets/icons/manifest.png\",\"large_text\":\"Editing a MANIFEST file\",\"small_image\":\"https://raw.githubusercontent.com/LeonardSSH/vscord/main/assets/icons/vscode.png\",\"small_text\":\"Visual Studio Code\"}},\"pid\":137344,\"socketId\":\"137344\"}"
[2024-12-22 01:51:02] [Process Scanner] Process scan triggered
[2024-12-22 01:51:08] [Process Scanner] Process scan complete
[2024-12-22 01:51:08] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:51:18] [Process Scanner] Process scan triggered
[2024-12-22 01:51:25] [Process Scanner] Process scan complete
[2024-12-22 01:51:25] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:51:25] [IPC] Incoming stream...
[2024-12-22 01:51:25] [IPC] Recieved message: {"v":1,"client_id":"212"}
[2024-12-22 01:51:25] [IPC] Recieved handshake
[2024-12-22 01:51:26] [IPC] Recieved message: {"args":{"pid":137808,"activity":{"details":"test","timestamps":{"start":1734828685597},"assets":{},"type":0,"instance":false}},"cmd":"SET_ACTIVITY","nonce":"1"}
[2024-12-22 01:51:26] [Client Connector] Sending payload for IPC activity: "{\"activity\":{\"type\":0,\"timestamps\":{\"start\":1734828685597000},\"application_id\":\"212\",\"details\":\"test\",\"assets\":{}},\"pid\":137808,\"socketId\":\"137808\"}"
[2024-12-22 01:51:35] [Process Scanner] Process scan triggered
[2024-12-22 01:51:41] [Process Scanner] Process scan complete
[2024-12-22 01:51:41] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:51:48] [IPC] Recieved message: {"args":{"pid":137808,"activity":{"details":"test","timestamps":{"start":1733182371000},"assets":{},"type":0,"instance":false}},"cmd":"SET_ACTIVITY","nonce":"2"}
[2024-12-22 01:51:48] [Client Connector] Sending payload for IPC activity: "{\"activity\":{\"type\":0,\"timestamps\":{\"start\":1733182371000000},\"application_id\":\"212\",\"details\":\"test\",\"assets\":{}},\"pid\":137808,\"socketId\":\"137808\"}"
[2024-12-22 01:51:51] [Process Scanner] Process scan triggered
[2024-12-22 01:51:57] [IPC] Recieved message: {"args":{"pid":137808,"activity":{"details":"test","assets":{},"type":0,"instance":false}},"cmd":"SET_ACTIVITY","nonce":"3"}
[2024-12-22 01:51:57] [Client Connector] Sending payload for IPC activity: "{\"activity\":{\"type\":0,\"application_id\":\"212\",\"details\":\"test\",\"assets\":{}},\"pid\":137808,\"socketId\":\"137808\"}"
[2024-12-22 01:51:58] [Process Scanner] Process scan complete
[2024-12-22 01:51:58] [Client Connector] Already sent payload for activity: DSX
[2024-12-22 01:52:08] [Process Scanner] Process scan triggered
```
appid: 212 is CustomRP and in the first two entries the time is set manually, in the last one I've removed it (should be handled by rsRPC/Discord)

**Comment by SpikeHD** [comment]
Timestamps should be fixed as of 9a7dfb73af4d8b463b3263d8e16d1c0d0eca7da2

**Comment by maicol07** [comment]
Works! Thank you!
